packages/venia-concept/src/components/Form/form.js

```diff
@@ -0,0 +1,29 @@
+import { createElement, Component } from 'react';
```

Comment:
This is a great, sleek form component, but since https://github.com/magento-research/pwa-studio/commit/8f387edf83c5416a9e379e1be5557bda0d12309a is already in the codebase, we should switch to using `informed` here. It shouldn't be a big change, because you've factored out the base form logic fairly well.

I know this is a hassle, but as we do the heavy lifting to get our code into 1.x state, there are some functionality collisions and this is one of them.

Reply:
No worries - it was easy to replace!